File: BCC221-POO/TPC++/classes_cpp/cliente.cpp
using namespace std;

#include "../headers/cliente.hpp"



Cliente::Cliente(string nome, long int cpf, int id, Ordem* ordem, Veiculo* veiculo) {

    this->nome = nome;
    this->cpf = cpf;
    this->id = id;
    ordem_servico = ordem;
    vei_cliente = veiculo;

}

long int Cliente::getCpf(){
    return this->cpf;
}

int Cliente::getId(){
    return this->id;
}

string Cliente::getNome(){
    return this->nome;
}

Ordem* Cliente::getOrdem(){
    return this->ordem_servico;
}

Veiculo* Cliente::getVeiculo(){
    return this->vei_cliente;
}

File: BCC221-POO/TPJAVA/TPJava/src/tpjava/Cliente.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tpjava;

/**
 *
 * @author nicolas
 */
public class Cliente {
    private String nome;
    private final double  CPF;
    private int idCliente;
    private Ordem ordemServico;
    private Veiculo veiculoCliente;

    public Cliente(String nome, double CPF, Ordem ordemServico, Veiculo veiculoCliente) {
        this.nome = nome;
        this.CPF = CPF;
        this.ordemServico = ordemServico;
        this.veiculoCliente = veiculoCliente;
    }



    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Ordem getOrdemServico() {
        return ordemServico;
    }

    public void setOrdemServico(Ordem ordemServico) {
        this.ordemServico = ordemServico;
    }

    public Veiculo getVeiculoCliente() {
        return veiculoCliente;
    }

    public void setVeiculoCliente(Veiculo veiculoCliente) {
        this.veiculoCliente = veiculoCliente;
    }
    
    public int getId(){
        return this.idCliente;
    }
    
    public void setId(int id){
        this.idCliente = id;
    }
    
    public double getCPF(){
        return this.CPF;
    }
    
    
}
